lib/components/viewers/trip-viewer.js

```diff
@@ -137,9 +161,8 @@ class TripViewer extends Component {
 const mapStateToProps = (state, ownProps) => {
   const viewedTrip = state.otp.ui.viewedTrip
   return {
-    languageConfig: state.otp.config.language,
-    timeFormat: coreUtils.time.getTimeFormat(state.otp.config),
     tripData: state.otp.transitIndex.trips[viewedTrip.tripId],
+    use24HourFormat: state.user?.loggedInUser?.use24HourFormat ?? false,
```

Comment:
Do we still need this prop?

Reply:
Nope, removed in https://github.com/opentripplanner/otp-react-redux/pull/480/commits/2ccefb263ad8863edab72e9d796596beda7336f0